WSL/file-permissions.md

```diff
@@ -54,11 +53,11 @@ If the file has metadata present, we simply use those metadata values instead of
 
 The result depends on if the file already has existing metadata.
 
-##### **The file does not have metadata (default)**
+##### chmod file does not have metadata (default)

```

Comment:
I'd revert this one back. To be accurate it's the file that you are using 'chmod' on. Perhaps another wording could be clearer here.

```suggestion
##### The file does not have metadata (default)

```